CHANGELOG.md

```diff
@@ -44,6 +44,7 @@
 - [#2357](https://github.com/oauth2-proxy/oauth2-proxy/pull/2357) Update ojg to latest release (@bitfehler)
 - [#1922](https://github.com/oauth2-proxy/oauth2-proxy/pull/1922) Added support for env variables in the alpha struct (@hevans-dglcom)
 - [#2235](https://github.com/oauth2-proxy/oauth2-proxy/pull/2235) Bump golang to 1.21 and min allowed version to 1.20 (@tuunit)
+- [#2451](https://github.com/oauth2-proxy/oauth2-proxy/pull/2451) Add support for prefix in SecretSource (@aslafy-z)
```

Comment:
Please add to section Since v7.6.0

Reply:
done with [`775b968` (#2451)](https://github.com/oauth2-proxy/oauth2-proxy/pull/2451/commits/775b96831e89fc8505b6797ea1027d8e1d09279f)